runner/runner.go

```diff
@@ -131,11 +131,15 @@ func (r *Runner) Run(ctx context.Context, args []string) int {
 		(!config.IsSet("WORKSPACE_NAMESPACE") || strings.Contains(config.GetString("WORKSPACE_NAMESPACE", ""), "free")) {
 		config.Set("statsExcludedTags", []string{"workspaceId", "sourceID", "destId"})
 	}
-	stats.Default = stats.NewStats(config.Default, logger.Default, svcMetric.Instance,
+	statsOptions := []stats.Option{
 		stats.WithServiceName(r.appType),
 		stats.WithServiceVersion(r.releaseInfo.Version),
 		stats.WithDefaultHistogramBuckets(defaultHistogramBuckets),
```

Comment:
For `defaultHistogramBuckets` we might need 2 more, `0.002` for really fast operations & `20` for slower ones but not slower than 1 min.

Reply:
I'm trying to decrese the buckets if possible :sweat_smile: 
What do you think are the use cases for the `0.002` and `20` buckets?

Reply:
https://rudderlabs.slack.com/archives/C01HTT66UMB/p1687271131609569?thread_ts=1687161567.768049&cid=C01HTT66UMB